HeliPort/NetworkInfo.swift

```diff
@@ -249,11 +268,17 @@ class NetworkManager {
                     "cut -d\\  -f2"
                 ).trimmingCharacters(in: .whitespacesAndNewlines)
                 if mac == matchingMAC {
-                    ipAndRouter.0 = NetworkManager.runCommand("ipconfig getifaddr \(interfaceArray[int])")
+                    let localIp = NetworkManager.runCommand("ipconfig getifaddr \(interfaceArray[int])")
```

Comment:
```swift
    class func getAddress(bsd: String) -> String? {
        var address: String?

        // Get list of all interfaces on the local machine:
        var ifaddr: UnsafeMutablePointer<ifaddrs>?
        guard getifaddrs(&ifaddr) == 0 else { return nil }
        guard let firstAddr = ifaddr else { return nil }

        // For each interface ...
        for ifptr in sequence(first: firstAddr, next: { $0.pointee.ifa_next }) {
            let interface = ifptr.pointee

            // Check for IPv4 or IPv6 interface:
            let addrFamily = interface.ifa_addr.pointee.sa_family
            if addrFamily == UInt8(AF_INET) || addrFamily == UInt8(AF_INET6) {

                // Check interface name:
                let name = String(cString: interface.ifa_name)
                if name == bsd {

                    // Convert interface address to a human readable string:
                    var hostname = [CChar](repeating: 0, count: Int(NI_MAXHOST))
                    getnameinfo(interface.ifa_addr, socklen_t(interface.ifa_addr.pointee.sa_len),
                                &hostname, socklen_t(hostname.count),
                                nil, socklen_t(0), NI_NUMERICHOST)
                    address = String(cString: hostname)
                }
            }
        }
        freeifaddrs(ifaddr)

        return address
    }
```

I found this solution from StackOverflow https://stackoverflow.com/a/30754194

Reply:
Would you recommend me using this source code instead of using ifconfig? Mine iterates through all mac addresses available and matches with the mac address from itlwm device info and calls ifconfig on that interface to get the ip address.

Reply:
Seems to be working fine with that code, however it does return an ipV6 address rather than an ipV4. Not sure if you are fine with that.

Reference:
![Screen Shot 2020-06-30 at 11 52 46 PM](https://user-images.githubusercontent.com/16653389/86204322-d681b000-bb2c-11ea-9c9d-8cab58218237.png)

Reply:
I think we could change this line from
```swift
if addrFamily == UInt8(AF_INET) || addrFamily == UInt8(AF_INET6) {
```
to
```swift
if addrFamily == UInt8(AF_INET) {
```
to ensure we get the IPv4 address, or maybe we should prefer IPv4 over IPv6 since it's possible that one could connect to an IPv6 only network.

Reply:
Yeah I will add a conditional statement to check if there is an ipv4 and if there isn’t then to use the ipv6